index.css

```diff
@@ -411,7 +520,8 @@ li {
 .project-description {
   margin-top: 13px;
   width: 95%;
-  max-width: 640px;
+
+  /* max-width: 640px; */
   font-family: "Poppins", sans-serif;
```

Comment:
[OPTIONAL] You can follow DRY principles for CSS and decrease the number of repeating elements. Here this line repeats 16 times in your CSS file and there is only one font used on your template. So you can define the elements in the beginning of your CSS file like p, a, span { font-family: font} 👍